packages/patterns-react/src/patterns/CardWithImages/index.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export { default as CardsWithImages } from './CardsWithImages';
```

Comment:
@mkothur you also need to export from `src/patterns/index.js`:

```
export * from './CardWithImages`;
```

Also, shouldn't the folder be called `CardsWithImages` and not `CardWithImages`?

Reply:
Okay